src/views/pages/network.html

```diff
@@ -53,74 +53,94 @@
 			<h1 class="page-title">Network</h1>
 
 			<div class="network-controls">
-				{{#each providers}}
-					<div class="network-control">
-						<input class="switch-inline {{this}}"
-							type="checkbox"
-							id="provider-{{this}}"
-							checked="{{enabled[@key]}}"
-							twoway="false">
-							<label for="provider-{{this}}" on-click="@this.toggleMap(@key)"></label>{{this}}
-					</div>
+				{{#each providers:idx}}
```

Comment:
Don't understand the goal of the pairing here. It'll now break if we have an odd number of providers. The previous version handled that case.

Reply:
Why it will break, it will render every pair of the array for odd and even number of elements. I paired em because the length of providers could be different and in this way it easy to style this block of controls properly with no need of manual fixing every time

Reply:
![image](https://user-images.githubusercontent.com/6192491/185921278-768bd513-cb97-49cb-a1fc-769d239d2f33.png)

Reply:
Correct (actually I think the last line was supposed to be centered in this case but we no longer have that screen in figma):

![image](https://user-images.githubusercontent.com/6192491/185921494-076366d8-187f-4eef-9078-e32575095002.png)

Reply:
So how it should look like?
 If the number of elements is odd, how should the last one line look? The control which takes the whole length of block and justified to the left? Or justified to the center, or in some other way?

Reply:
Like the last picture but probably centered.

Reply:
Ok

Reply:
![1](https://user-images.githubusercontent.com/13244319/185930514-515fa05b-a39c-4834-83df-fe26ff6bccf9.png)
Like this right?

Reply:
yes